SU-HSDO/suhumsci
# STN-540:  Experimental setting and raised card padding

# READY FOR REVIEW

## Summary
Develop a theme setting toggle for experimental features. 
Add refinement of hb-raised cards padding with the experimental feature in the Colorful theme.

JIRA Card: [STN-540](https://sparkbox.atlassian.net/browse/STN-540)

## Need Review By (Date)
10/26

## Urgency
medium

## Steps to Test
1. Run `npm run test`
2. To verify and turn on the experimental feature setting, go to: (Appearance > (Any Humsci Basic Theme) > Settings > Experimental Features)
3. Check the experimental feature checkbox and hit save.
4. You can now verify that the experimental features class `hb-experimental` is active on the `<html>` tag. 
5. To test the hb-raised-card padding refinement, you need to have the Colorful theme and experimental setting enabled.
7. Once you have those appearance settings set, go to http://sparkbox-sandbox.suhumsci.loc/qa/qa-raised-postcards
8. This page will have the padding changes applied to both horizontal and vertical raised postcards of the top two sections, raised vertical linked cards have their own padding setting and should not be affected.
9. If you do not see 4 columns on the second section of vertical cards, to test against what is in the Jira card, you'll want to add another card column.
10. Verify the padding for those top two raised card sections have slimmer padding of 2.4rem, rather than 4.8rem.
11. Make sure when you switch the experimental setting back off the padding goes back to it's normal non-experimental padding size, this is most obvious at (lg) desktop size.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
